sdk/iOS/src/MSUser.h

```diff
@@ -40,7 +40,7 @@
 /// requests made to the Microsoft Azure Mobile Service, allowing the client to
 /// perform all actions on the Microsoft Azure Mobile Service that require
 /// authenticated-user level permissions.
-@property (nonatomic, copy)         NSString *mobileServiceAuthenticationToken;
+@property (nonatomic, copy, nonnull)         NSString *mobileServiceAuthenticationToken;
```

Comment:
In the serialisation of the user in [MSLoginSerializer](https://github.com/Azure/azure-mobile-services/blob/MobileApp/sdk/iOS/src/MSLoginSerializer.m#L92-L103) the user isn't construct without the userId and authToken being NSString instances. Hence why I thought they should be nonnullable. Also as with the logout, it just sets the currentUser to nil rather than specific properties